### PR TITLE
fix: Add sachinparihar

### PR DESCRIPTION
**Pull request by sachinparihar**
resolves #issueNumber

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [x] Changes made in this PR are mentioned below
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.
- [x] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/sachinparihar.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "image": "https://user-images.githubusercontent.com/94243074/183848297-0cd1bd02-e62d-4ce0-90ec-3d557a394111.png",
+    "issueId": 1292,
+    "name": "Sachin Parihar",
+    "username": "sachinparihar"
+  }
```
Comment (Pradumnasaraf) [inline profiles/sachinparihar.json:6]:
```suggestion
{
  "image": "https://user-images.githubusercontent.com/94243074/183848297-0cd1bd02-e62d-4ce0-90ec-3d557a394111.png",
  "issueId": 1292,
  "name": "Sachin Parihar",
  "username": "sachinparihar"
  }
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey, 
Everything looks good, just minor indentation issues.

File: profiles/sachinparihar.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "image": "https://user-images.githubusercontent.com/94243074/183848297-0cd1bd02-e62d-4ce0-90ec-3d557a394111.png",
+  "issueId": 1292,
+  "name": "Sachin Parihar",
+  "username": "sachinparihar"
+  }
+
```
Comment (Panquesito7) [inline profiles/sachinparihar.json:7]: There's an extra space here. GitHub doesn't display the endline.
```suggestion
}
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Reusable

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing! 🎉


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Pull request merged**